frontend/package.json

```diff
@@ -6,10 +6,16 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "axios": "^0.21.1",
+    "bootstrap": "^4.6.0",
```

Comment:
Az index.html fájlban be van húzva cdn-ről a bootstrap, akkor szerintem csomagként nem szükséges felrakni.